bin/push.bash

```diff
@@ -1,65 +0,0 @@
-#!/bin/bash -x
```

Comment:
Yep, it's just running docker build. We can't have automatic tagging but we can also have a 'stable' tag or something.